content/pt/ecosystem/integrations.md

```diff
@@ -74,4 +74,4 @@ observabilidade aos usuários finais, consulte
 
 [envie um PR]: /docs/contributing/pull-requests/
 
-{{% ecosystem/keep-up-to-date integration %}}
+{{% include keep-up-to-date.md integration %}}
```

Comment:
@open-telemetry/docs-pt-approvers @vitorvasc - I'll let you update the included word (here and in the other `pt` pages) name when you localize `keep-up-to-date.md`.